package/MDAnalysis/coordinates/TRJ.py

```diff
@@ -693,7 +813,7 @@ class NCDFWriter(base.WriterBase):
         File "<string>", line 1, in <module>
         File "/scratch2/miniconda/envs/py35/lib/python3.5/site-packages/netCDF4/__init__.py", line 3, in <module>
           from ._netCDF4 import *
-      ImportError: /scratch2/miniconda/envs/py35/lib/python3.5/site-packages/netCDF4/_netCDF4.cpython-35m-x86_64-linux-gnu.so: undefined symbol: nc_inq_var_fletcher32
+      ImportError: /scratch2/minicthis will be changedonda/envs/py35/lib/python3.5/site-packages/netCDF4/_netCDF4.cpython-35m-x86_64-linux-gnu.so: undefined symbol: nc_inq_var_fletcher32
```

Comment:
This looks odd... text inserted wrongly??

Reply:
My bad, seems like an erroneous middle mouse button click. Fixed in 
4b12592.